azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS image_types (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  type             VARCHAR(64) NOT NULL UNIQUE,
+  description      VARCHAR(2048),
+  active           BOOLEAN,
+  deployable       VARCHAR(64),
+  created_on       TIMESTAMP       NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  created_by       VARCHAR(64)     NOT NULL,
+  modified_on      TIMESTAMP       DEFAULT NULL,
+  modified_by      VARCHAR(64)
+);
+
+CREATE TABLE IF NOT EXISTS image_versions (
+  id               INT         NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  path             VARCHAR(1024) NOT NULL,
+  description      VARCHAR(2048),
+  version          VARCHAR(64) NOT NULL,
+  type_id          INT NOT NULL, FOREIGN KEY(type_id) references image_types (id),
```

Comment:
I personally like reference name as table_name_id. In this case, it should be image_types_id (image_type_id if you rename image_types table). With that, table name of image_types should be renamed as image_type. Plural should not be used for table name.

Reply:
Keeping just id as the primary key makes it simple. Following the same convention for all the tables and making them  plural such as image_types, image_versions, image_ownerships  etc. So, this goes well with all the different entities involved in image management. This s similar to existing tables such as execution_flows, execution_logs etc.